File: api/src/main/java/ru/romanow/protocols/api/model/TestObjectResponse.java
package ru.romanow.protocols.api.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;
import java.io.Serializable;

@Data
@Accessors(chain = true)
@AllArgsConstructor
@NoArgsConstructor
@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class TestObjectResponse
        implements Serializable {

    @XmlElement
    private Integer code;

    @XmlElement
    private String data;
}

File: api/src/main/java/ru/romanow/protocols/api/model/TestObjectResponse.kt
package ru.romanow.protocols.api.model

import java.io.Serializable
import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
data class TestObjectResponse(
    @XmlElement var code: Int? = null,
    @XmlElement var data: String? = null
) : Serializable